dolphinscheduler-spi/src/test/java/org/apache/dolphinscheduler/spi/utils/JSONUtilsTest.java

```diff
@@ -17,6 +17,9 @@
 
 package org.apache.dolphinscheduler.spi.utils;
 
+import org.apache.dolphinscheduler.common.utils.DateUtils;
```

Comment:
Maybe also merge this tests to the common module? It's weird `JSONUtils` is in common module but `JSONUtilsTest` is in spi module

Reply:
OK, I will update it.

Reply:
done